app/representers/member_permissions_representer.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# This representer represents member permissions in a user-friendly way
+# Examples:
+# --------------------------------------------------
+# XML:
+# --------------------------------------------------
+# <permissions>
+#   <user_id>10</user_id>
+#   <role>member</role>
+#   <allowed_sections>
+#     <admin_section>portal</admin_section>
+#     <admin_section>monitoring</admin_section>
+#     <admin_section>settings</admin_section>
+#   </allowed_sections>
+#   <allowed_service_ids>
+#     <allowed_service_id>2</allowed_service_id>
+#     <allowed_service_id>3</allowed_service_id>
+#   </allowed_service_ids>
+# </permissions>
+# --------------------------------------------------
+# JSON:
+# --------------------------------------------------
+# {
+#   "permissions": {
+#     "user_id": 10,
+#     "role": "member",
+#     "allowed_service_ids": [ 2, 3 ],
+#     "allowed_sections": [ "portal", "monitoring", "settings" ],
+#     "links": [{
+#         "rel": "user",
+#         "href": "http://provider-admin.3scale.net/admin/api/accounts/2/users/10"
+#     }]
+#   }
+# }
+
+class MemberPermissionsRepresenter < ThreeScale::Representer
+
+  wraps_resource :permissions
+
+  property :user_id, getter: ->(opts) { opts[:user].id }
+  property :role, getter: ->(opts) { opts[:user].role }
+
+  collection :allowed_sections, getter: ->(opts) { opts[:user].allowed_sections }
+  collection :allowed_service_ids, getter: ->(opts) { opts[:user].allowed_service_ids }, render_nil: true
+
+  class JSON < MemberPermissionsRepresenter
+    include Roar::JSON
+
+    link :user do |opts|
+      admin_api_account_user_url(opts[:user].account, opts[:user].id)
```

Comment:
Minor detail:
```ruby
user = opts[:user]
admin_api_account_user_url(user.account_id, user)
```
So we don't fetch `opts[:user]` twice (although it may be pretty quick anyway) and we don't need to get the whole `account` only for this, with the `account_id` is enough 😄